lib/src/main/kotlin/com/lemonappdev/konsist/api/ext/list/KoRepresentsTypeProviderListExt.kt

```diff
@@ -55,15 +52,12 @@ fun <T : KoRepresentsTypeProvider> List<T>.withRepresentedTypeOf(kClass: KClass<
  * @param kClasses The Kotlin classes representing the types to exclude.
  * @return A list containing declarations without types matching the specified Kotlin classes.
  */
-fun <T : KoRepresentsTypeProvider> List<T>.withoutRepresentedTypeOf(kClass: KClass<*>, vararg kClasses: KClass<*>): List<T> =
+fun <T : KoRepresentsTypeProvider> List<T>.withoutRepresentedTypeOf(kClass: KClass<*>?, vararg kClasses: KClass<*>?): List<T> =
     filter {
-        kClass.qualifiedName?.let { type -> !it.representsType(type) } ?: true &&
+        kClass?.qualifiedName?.let { type -> !it.representsType(type) } ?: true &&
```

Comment:
Please replace with `it.representsType(kClass?.qualifiedName)`

Reply:
amended on this commit: https://github.com/LemonAppDev/konsist/pull/719/commits/c8bc914bc6cd845ad25229f0b25bf106a2304d3f.

All of them would be squashed together during merge, but are currently split for easier code review.